cmd/agent/app/integrations_test.go

```diff
@@ -142,3 +142,35 @@ func TestNormalizePackageName(t *testing.T) {
 	assert.Equal(t, normalizePackageName("datadog-checks_base"), "datadog-checks-base")
 	assert.Equal(t, normalizePackageName("datadog_checks_downloader"), "datadog-checks-downloader")
 }
+
+func TestParseWheelPackageNameValidCases(t *testing.T) {
+	tests := map[string]struct {
+		wheelFileName string
+		expectedName  string
+	}{
+		"name as first line":  {"datadog_cisco_aci_name_first_line_valid.whl", "datadog-cisco-aci-1"},
```

Comment:
nit: can't we call these wheels something (ie. without `cisco` on them)?

Reply:
👍 fixed.